package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@grammyjs/commands",
-    "version": "0.9.0",
+    "version": "0.10.0",
```

Comment:
This would skip a release. Last release was 0.8.1
```suggestion
"version": "0.9.0",
```

Reply:
Reasons not to mix feature development and release management

Reply:
let's revert this and do it all on main :)